Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -789,6 +882,13 @@ else if (mPreferences.getInt("wiiController", 3) == 4)
       else
         wiiClassicPortraitDefaultOverlay();
     }
+    else if (mPreferences.getInt("wiiController", 3) == 5)
+    {
+      if (isLandscape)
+        DrumsDefaultOverlay();
+      else
+        DrumsPortraitDefaultOverlay();
```

Comment:
These two methods should start with lowercase letters to follow our Java coding style.